src/VirtualClient/VirtualClient.Main/profiles/PERF-GPU-MLPERF-TRAINING.json

```diff
@@ -0,0 +1,83 @@
+{
+    "Description": "MLPerf Training GPU Performance Workload",
+    "Metadata": {
+        "RecommendedMinimumExecutionTime": "10:00:00",
+        "SupportedPlatforms": "linux-x64",
+        "SupportedOperatingSystems": "Ubuntu",
+        "SpecialRequirements": "This is an Nvidia GPU-specialized workload. It depends upon the system having an Nvidia GPU card/chip."
+    },
+    "Parameters": {
+        "Username": "azureuser",
```

Comment:
Do we want user to be defined by default here? To get current username, this should be empty, right?

Reply:
Updated.